src/main/java/lotto/domain/LottoNumberPool.java

```diff
@@ -0,0 +1,29 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumberPool {
+
+    private static final List<LottoNumber> lottoNumbers = new ArrayList<>();
+
+    private LottoNumberPool() {
+    }
+
+    static {
+        IntStream.rangeClosed(1, 45)
+                 .mapToObj(LottoNumber::new)
+                 .forEach(lottoNumbers::add);
+    }
+
+    public static List<LottoNumber> getLottoNumbers() {
+        Collections.shuffle(lottoNumbers);
+        return lottoNumbers.stream()
+                           .limit(6)
+                           .map(LottoNumber::new)
+                           .collect(Collectors.toList());
```

Comment:
`List`의 `subList()` 메서드를 활용해볼 수도 있는 점 단순 참고차 말씀드려봅니다.
```suggestion
        return new ArrayList<>(lottoNumbers.subList(0, 6));
```

Reply:
공유 감사드립니다 :) 

Immutable 목록을 반환하기 위해서 subList에서 stream의 limit로 변경했습니다.